api/HTMLMediaElement.json

```
@@ -3093,10 +3105,10 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/media.html#event-media-progress",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "32"
```

This also seems too late.

---

Yep, it was in the initial WebKit support:
https://trac.webkit.org/browser/webkit/trunk/WebCore/html/HTMLMediaElement.cpp?rev=27277#L452

```suggestion
              "version_added": "1"
```